### PR TITLE
change path to fix windows log

### LINKED ISSUE #5680: ibmcloud command failing on windows
/area testing

I've noticed that `ibmcloud` command that is being executed as part of our test suite on windows tests are failing 

https://cloud.ibm.com/devops/pipelines/929da3ed-6cad-4026-a847-7a8485f20fa4/582614a3-b910-4659-ac27-6288668c5fe6/a1d74ffa-3d39-4dbb-8583-b0a34f87f3bd?env_id=ibm:yp:eu-de

This is probably due to the use of POSIX style path and not using windows path

```
+ ibmcloud cos upload --bucket odo-tests-openshift-logs --key pr-5675-windows-tests-651.html --file /tmp/pr-5675-windows-tests-651.html
FAILED
The value in flag '--file' is invalid
NAME: 
   ibmcloud cos upload - Upload objects from S3 concurrently.

USAGE:
   ibmcloud cos upload --bucket BUCKET_NAME --key KEY --file PATH [--concurrency value] [--max-upload-parts PARTS] [--part-size SIZE] [--leave-parts-on-errors] [--cache-control CACHING_DIRECTIVES] [--content-disposition DIRECTIVES] [--content-encoding CONTENT_ENCODING] [--content-language LANGUAGE] [--content-length SIZE] [--content-md5 MD5] [--content-type MIME] [--metadata STRUCTURE] [--region REGION] [--output FORMAT] [--json]

OPTIONS:
   --bucket BUCKET_NAME                 The name (BUCKET_NAME) of the bucket.
   --key KEY                            The KEY of the object.
   --file PATH                          The PATH to the file to upload.
   --concurrency value                  The number of goroutines to spin up in parallel per call to Upload when sending parts. Default value is 5.
   --max-upload-parts PARTS             Max number of PARTS which will be uploaded to S3 that calculates the part size of the object to be uploaded.  Limit is 10,000 parts.
   --part-size SIZE                     The buffer SIZE (in bytes) to use when buffering data into chunks and ending them as parts to S3. The minimum allowed part size is 5MB.
   --leave-parts-on-errors              Setting this value to true will cause the SDK to avoid calling AbortMultipartUpload on a failure, leaving all successfully uploaded parts on S3 for manual recovery.
   --cache-control CACHING_DIRECTIVES   Specifies CACHING_DIRECTIVES for the request/reply chain.
   --content-disposition DIRECTIVES     Specifies presentational information (DIRECTIVES).
   --content-encoding CONTENT_ENCODING  Specifies what content encodings (CONTENT_ENCODING) have been applied to the object and thus what decoding mechanisms must be applied to obtain the media-type referenced by the Content-Type header field.
   --content-language LANGUAGE          The LANGUAGE the content is in.
   --content-length SIZE                SIZE of the body in bytes. This parameter is useful when the size of the body cannot be determined automatically.
   --content-md5 MD5                    The base64-encoded 128-bit MD5 digest of the data.
   --content-type MIME                  A standard MIME type describing the format of the object data.
   --metadata STRUCTURE                 A STRUCTURE using JSON syntax. See IBM Cloud Documentation.
   --region REGION                      The REGION where the bucket is present. If this flag is not provided, the program will use the default option specified in config.
   --output FORMAT                      Output FORMAT can be only json or text.
   --json                               [Deprecated] Output returned in raw JSON format.

```

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this:**

/kind tests

**What does this PR do / why we need it:**
This PR will fix the broken link to the logs for the windows CI tests. 
**Which issue(s) this PR fixes:**

Fixes #5680

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

**Comment by dharmit** [comment]
> Fixes #

Is it #5681 that this PR will fix?

**Comment by anandrkskd** [comment]
> Is it https://github.com/redhat-developer/odo/issues/5681 that this PR will fix?

No Dharmit, that issue is related `ginkgo`. This issue will fix #5680

Fixes https://github.com/redhat-developer/odo/issues/5680

**Comment by dharmit** [comment]
> > Is it #5681 that this PR will fix?
> 
> No Dharmit, that issue is related `ginkgo`. This issue will fix #5680
> 
> Fixes #5680

OK, thanks. 

Btw, putting `Fixes #<issue-number>` technique works only when you put it in the PR description. Saying based on my observation on this very PR. 😉

**Comment by anandrkskd** [comment]
> Btw, putting Fixes #<issue-number> technique works only when you put it in the PR description. Saying based on my observation on this very PR. 

I guess it worked for me 😸

**Review by feloy — COMMENTED** [review]
Sorry, I'm sending these comments a little late

File: .ibm/pipelines/windows-test.sh [hunk 1]
```diff
@@ -20,7 +20,7 @@ echo "RESULT: $RESULT"
 # save log
 source .ibm/pipelines/functions.sh
 ibmcloud login --apikey "${API_KEY}" -r "${IBM_REGION}"
-sshpass -p $WINDOWS_PASSWORD scp -o StrictHostKeyChecking=no Administrator@$WINDOWS_IP:/tmp/${LOGFILE} /tmp/${LOGFILE}
+sshpass -p $WINDOWS_PASSWORD scp -o StrictHostKeyChecking=no Administrator@$WINDOWS_IP:~/AppData/Local/Temp/${LOGFILE} /tmp/${LOGFILE}
```
[the remark this answers is not in the document]
Comment (feloy) [reply .ibm/pipelines/windows-test.sh:23]: My bad, this script is running on the Linux side, right?

**Pull request merged**